tests/libstorage/storageclass.go

```diff
@@ -74,63 +74,7 @@ func GetSnapshotStorageClass(client kubecli.KubevirtClient) (string, error) {
 		return Config.StorageSnapshot, nil
```

Comment:
I would check here that the provided storage class, actually supports snapshots, most of which can be stoled from the code below

Reply:
Yeah, we can, done